website/static/static/index.css

```diff
@@ -253,3 +320,21 @@ b {
 .svg-inline--fa.fa-w-16 {
     width: 1em;
 }
+
+@keyframes slide-right {
+    100% { transform: translateX(0%); }
+}
+
+@-webkit-keyframes slide-right {
+    100% { -webkit-transform: translateX(0%); }
+}
+
+@keyframes slide-left {
+    0% { transform: translateX(0%); }
+    100% { transform: translateX(-110%); }
+}
+
+@-webkit-keyframes slide-left {
+    0% { -webkit-transform: translateX(0%); }
+    100% { -webkit-transform: translateX(-110%); }
+}
```

Comment:
I wonder if we need `-moz-` versions too.  I checked Firefox and it works as is, but my FF is pretty recent.

Reply:
@kevinbarabash FireFox >= 16 supports without the prefix: https://caniuse.com/#feat=css-animation